src/support/kuzzle.ts

```diff
@@ -227,9 +227,12 @@ export class KuzzleSDK {
 
     // Send the request
     return new Promise((resolve, reject) => {
-      const req = http.request(url, options, (res) => {
-        resolve(res);
-      });
+      const req = this.ssl ?
```

Comment:
I see we resolved the same problem :smile: 

Thus, instead of repeating the code, you can just assign the module before using it `const httpModule = this.ssl ? https : http;`

Reply:
Didn't see that error comming while we developed this :sweat_smile: 

Yea sure, I'll update this is not elegant coding I admit, I did it a little to fast